source/gcp/supported-services/cloud_audit_logs.rst

```diff
@@ -5,6 +5,10 @@
 Audited resources
 =================
 
+.. note::
+    This service is supported by Wazuh using the **gcp-pubsub** module. Details on how to configure this module can be found :ref:`here <gcp-pubsub>`.
+
+
 Google Cloud maintains three `audit logs <https://cloud.google.com/monitoring/api/resources#tag_audited_resource>`__ for each Google Cloud project, folder and organization: **Admin Activity**, **Data Access** and **System Event**.
 
 .. thumbnail:: ../../images/gcp/gcp-overview.png
```

Comment:
I've added all the requested changes except for this one. When I was trying to configure GCP to be able to pull the necessary logs to Wazuh to make new screenshots, I noticed that this section of the configuration is lacking any explanation about how to configure the services.
Since those services are out of the scope of this PR, I opened #4672 to rework the `source/gcp/supported-services/dns_queries.rst` and `source/gcp/supported-services/cloud_audit_logs.rst` files.

Please, tell me your thoughts on this approach.

Reply:
@noise-kngdm no worries. Sounds good!